src/main/features/response/tasks/yourRepaymentPlanTask.ts

```diff
@@ -0,0 +1,14 @@
+import { Validator } from 'class-validator'
+import { ResponseDraft } from 'response/draft/responseDraft'
+
+const validator = new Validator()
+
+function isValid (input): boolean {
```

Comment:
We have to extract this to the function at some point because that simple logic is repeated in all places. Not part of this story though.

Reply:
agree, i am going to leave it for now